File: w09/d01/amy_ruan/mailer/spec/mail_spec.rb
require_relative('../lib/mail')

describe Mail do
  it "has a ship_to_address" do
    mail = Mail.new(ship_to_address: "902 Broadway")
    expect(mail.ship_to_address).to eq "902 Broadway"
  end
  it "should have a stamp value" do
    mail = Mail.new(stamp_value: 500)
    expect(mail.stamp_value).to eq(500)
  end
end

File: w09/d03/edward_shin/mail_objects_app/spec/javascripts/mail_spec.js
describe("mail", function(){

  it("has a shipping address", function(){
    var mail = new Mail();
    mail.shippingAddress = "1 main st";
    expect(mail.shippingAddress).toBe("1 main st");
  });

  it("has a stamp value", function(){
    var mail = new Mail();
    mail.stampValue = 5;
    expect(mail.stampValue).toBe(5);
  });

})